specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2021-01-01/examples/AvailabilityStatuses_ListBySubscriptionId.json

```diff
@@ -0,0 +1,72 @@
+{
+  "parameters": {
+    "subscriptionId": "subscriptionId",
+    "api-version": "2015-01-01",
+    "$expand": "recommendedactions"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "<resourceId>/providers/Microsoft.ResourceHealth/AvailabilityStatueses/current",
```

Comment:
Fix the typo on the id: AvailabilityStatuses.

Reply:
irrelevant for current resource type registration, ack for next iteration.